cloud-minecraft/cloud-minecraft-extras/src/main/java/cloud/commandframework/minecraft/extras/MinecraftHelp.java

```diff
@@ -680,8 +701,8 @@ private Component formatDescription(final ArgumentDescription description) {
          * Creates a component from a command sender, key, and arguments
          *
          * @param sender command sender
-         * @param key message key (constants in {@link MinecraftHelp}
-         * @param args args
+         * @param key    message key (constants in {@link MinecraftHelp}
+         * @param args   args
```

Comment:
Are JD comments aligned? seems to be inconsistent across the parts of the codebase you've modified here

Reply:
In general yes, `@param` descriptions are aligned, this was the only non-aligned JD in this file. Looks like I did forget to align them for the Bukkit/Paper methods though